modules/ssh/manifests/extra_authorized_key.pp

```diff
@@ -0,0 +1,19 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+define ssh::extra_authorized_key($username, $key, $home, $from='', $command='') {
```

Comment:
resource definitions and classes that utilize parameters should ALWAYS include there explicit type.  eg. String, Integer, Optional, etc.  Each parameter should also be on its own line.